__tests__/accessiblity_audit.test.js

```diff
@@ -0,0 +1,130 @@
+/* eslint-env jest */
+const configPaths = require('../config/paths.json')
+const PORT = configPaths.testPort
+// const axe = require('../../../lib/axe-helper')
+const { AxePuppeteer } = require('axe-puppeteer')
+
+let browser
+let page
+let baseUrl = 'http://localhost:' + PORT
+
+beforeAll(async (done) => {
+  browser = global.browser
+  page = await browser.newPage()
+  done()
+})
+
+afterAll(async (done) => {
+  await page.close()
+  done()
+})
+
+describe('Accessibility Audit', () => {
+  describe('Home page - layout.njk', () => {
```

Comment:
This might be a good candidate for `describe.each` as most of the tests appear to be the same.

(Example of [describe.each in the Error Summary tests in Frontend](https://github.com/alphagov/govuk-frontend/blob/master/src/components/error-summary/error-summary.test.js#L46))

Reply:
I think that might put us in a tricky position if/when these tests fail with new additions to the D.S. since you'd have to split the tests back out?

Easy code to change in the future > some repetition

Reply:
I agree with @nickcolley, ATM there is only a few tests here so having the repetition feels better and easier to read at this stage.

Reply:
👍